include/cpp2util.h

```diff
@@ -887,6 +887,7 @@ class out {
     #endif
 #endif
 
+#define CPP2_UFCS_EMPTY(...)
```

Comment:
Maybe `CPP2_UFCS_EMPTY` wasn't necessary, and passing `CPP2_UFCS_IDENTITY` instead would have done the job.

Reply:
But then all the cases would be the same again, and we'd have the original problem in non-SFINAE cases?

Reply:
Ah, you're right.